packages/patternfly-4/react-core/src/components/Accordion/Accordion.md

```diff
@@ -254,3 +254,126 @@ class FixedAccordion extends React.Component {
   }
 }
 ```
+
+# Accordion definition list example
```

Comment:
I think we should render the initial examples as definition lists since that is the default, and the last new example should be setting it to false to render as the non-default div/h3 list...